usr/share/rear/lib/layout-functions.sh

```diff
@@ -590,3 +589,27 @@ is_disk_a_pv() {
 function is_multipath_path {
     [ "$1" ] && type multipath &>/dev/null && multipath -c /dev/$1 &>/dev/null
 }
+
+# retry_command () is binded with REAR_SLEEP_DELAY and REAR_MAX_RETRIES.
+# This function will do maximum of REAR_MAX_RETRIES command execution
+# and will sleep REAR_SLEEP_DELAY after each unsuccessful command execution.
+# Function returns command result as soon as it succeeded.
+# Function accepts two parameters:
+# 1. Message that should be displayed in case that command did not succeed in REAR_MAX_RETRIES times.
+# 2. Command that should be repeated REAR_MAX_RETRIES, or return on success
+function retry_command ()
+{
+    local retry=0
+
+    until command_stdout=$(eval "$@"); do
+        sleep $REAR_SLEEP_DELAY
+
+        let retry++
+
+        if (( retry >= REAR_MAX_RETRIES )) ; then
+            return 1
```

Comment:
Either `Log` the error message here or remove the function parameter declaration for the message.